src/sc/sidechain.h

```diff
@@ -78,6 +109,15 @@ namespace Sidechain {
     bool hasScCreationOutput(const CTransaction& tx, const uint256& scId);
 
     bool checkCertSemanticValidity(const CScCertificate& cert, CValidationState& state);
+
+
+    enum class state {
```

Comment:
This also should be added in the output of getscinfo RPC cmd

Reply:
Agree, but as discussed AFK,  let's make it subject on subsequent PR